removing sbt related stuff and adding pom.xml

lalottacontinua opened the pull request:
there is one error when executing mvn clean test

Commit:
removing sbt related stuff and adding pom.xml
there is one error when executing mvn clean test

thinkberg commented on pom.xml:
```diff
@@ -0,0 +1,301 @@
+    </distributionManagement>
+
+    <build>
+
```
hier sollte noch 
 ```
<sourceDirectory>${basedir}/src/main/scala</sourceDirectory>
        <testSourceDirectory>${basedir}/src/test/scala</testSourceDirectory>

```
rein

Commit:
fixing version to snapshort
adjusting new ubirch-config version
adding scoverage plugin
changing deployment and pull repositories
